extra/changelog-generator/changelog-generator

```diff
@@ -22,6 +22,19 @@ SHA_TO_TRACKER = {}
 JIRA_REGEX = r"(?:Jira:? *)?(?:https?://tracker.mender.io/browse/)?((?:INF|ARCHIVE|MEN|QA|MC)(?:-| )[0-9]+)"
 TRACKER_REGEX = r"\(?(?:Ref:? *)?%s\)?:? *" % (JIRA_REGEX)
 
+def aggregate_dependabot_changelogs(entry_list):
+    new_entry_list = []
+    aggregated_dependabot_changelogs = "Aggregated Dependabot Changelogs:\n"
+    for entry in entry_list:
+        if re.match("[Bb]umps? .* from .* to .*", entry):
```

Comment:
Question: Is this good enough as a matcher for dependabot commits?

Alternative is the `Signed-off-by` tag maybe?

Reply:
I think this is alright, it's pretty explicit about which kinds of messages it is processing.